packages/ui/src/utils/parse-seconds.ts

```diff
@@ -0,0 +1,18 @@
+export function parseSeconds(seconds: number): string {
```

Comment:
This is probably a dumb question because I'm not familiar with NN code base, but isn't this code duplicated from `packages/core/src/utils/index.ts`?

Reply:
Yes but the UI doesn't install the core module so needs to exist sin both places. Not a biggie IMO.